dist/Interfaces/role/Role.d.ts

```diff
@@ -1,3 +1,6 @@
 export default interface Role {
+    id?: string;
+    createdAt?: Date;
+    updatedAt?: Date;
```

Comment:
It's good to put `Date | string` just in case it throws TS errors

Reply:
will do!